### PR TITLE
Amendment of the "mathematical description"

### PR DESCRIPTION
**Pull request by olorinthewhite**
exceptions: errors 12,14,15,16

### CODE REVIEW

File: doc/theorydoc/storage_prob.rst [hunk 1]
```diff
@@ -80,12 +80,12 @@ the storages can occur. The commodity balance function is thus changed to:
 .. math::
    &\forall y\in Y,~v\in V,~c \in C,~t\in T_m:\\\\
    \text{CB}(y,v,c,t)=&
-   \sum_{(y,v,c,p)\in C^{\text{in}}_{y,v,c,p}}\epsilon^{\text{in}}_{yvcpt}+
+   \Bigg( \sum_{(y,v,c,p)\in C^{\text{in}}_{y,v,c,p}}\epsilon^{\text{in}}_{yvcpt}+
    \sum_{(y,v,s,c)\in C^{\text{in}}_{y,v,s,c}}\epsilon^{\text{in}}_{yvst}+
-   \sum_{(y,a,f)\in A^{\text{in}}_{v}}\pi^{\text{in}}_{yaft}-\\\\
+   \sum_{(y,a,f)\in A^{\text{in}}_{v}}\pi^{\text{in}}_{yaft}\\\\
```
Comment (olorinthewhite) [inline doc/theorydoc/storage_prob.rst:85]: new 

image

File: doc/implementdoc/equ-commodity.rst [hunk 2]
```diff
@@ -403,17 +403,17 @@ and calculated by the following code fragment:
...
-:math:`v\in V` and timesteps :math:`t \in t_m` must be less than or equal to
+:math:`v\in V` and timesteps :math:`t \in T_m` must be less than or equal to
 the parameter maximum global CO2 emission budget
 :math:`\overline{\overline{L}}_{CO_{2},y}`, where the amount of released CO2 in
 a single support timeframe :math:`y` in a single site :math:`v` and at a single
 timestep :math:`t` is calculated by the product of the commodity balance of
 environmental commodities :math:`\mathrm{CB}(y,v,CO_{2},t)` and the parameter
 weight :math:`w`. This constraint is skipped if the value of the parameter
-:math:`\overline{\overline{L}}_{CO_{2}}` is set to ``inf``. The mathematical
+:math:`\overline{\overline{L}}_{CO_{2},y}` is set to ``inf``. The mathematical
```
Comment (Dthushara246) [inline doc/implementdoc/equ-commodity.rst:416]: Global CO2 Budget is '\overline{\overline{L}}_{CO_{2}}'?

File: doc/implementdoc/param-tech.rst [hunk 3]
```diff
@@ -102,7 +102,7 @@ Technical Parameters
     +---------------------------------------------+----+--------------------------------------------+
     |:math:`\overline{K}_{yaf}`                   |MW  |Transmission Capacity Upper Bound           |
     +---------------------------------------------+----+--------------------------------------------+
-    |:math:`T_{af}`                               |MW  |Remaining lifetime of installed transmission|
+    |:math:`T_{af}`                               |MW  |Remaining Lifetime of Installed Transmission|
```
Comment (Dthushara246) [inline doc/implementdoc/param-tech.rst:105]: Unit 'a'

File: doc/implementdoc/equ-commodity.rst (hunk 2 above)
Comment (olorinthewhite) [reply doc/implementdoc/equ-commodity.rst:416]: Hi, Global CO2 Budget is not a function of modeled years (y). It corresponds to "CO2 Emission Budget for modeling horizon".

grafik

File: doc/implementdoc/param-tech.rst (hunk 3 above)
Comment (olorinthewhite) [reply doc/implementdoc/param-tech.rst:105]: Yes, you are correct. It seems, I have missed this unit mistake.

File: urbs/model.py [hunk 4]
```diff
@@ -614,7 +614,7 @@ def def_partial_process_input_rule(m, tm, stf, sit, pro, coin):
     throughput_factor = (R - min_fraction * r) / (1 - min_fraction)
 
     return (m.e_pro_in[tm, stf, sit, pro, coin] ==
-            m.dt * m.cap_pro[stf, sit, pro] * online_factor +
+            m.dt * (m.cap_pro[stf, sit, pro] * online_factor +
             m.tau_pro[tm, stf, sit, pro] * throughput_factor)
```
Comment (Dthushara246) [inline urbs/model.py:618]: Another closing bracket at the end is needed.

File: urbs/model.py [hunk 5]
```diff
@@ -629,7 +629,7 @@ def def_partial_process_output_rule(m, tm, stf, sit, pro, coo):
     throughput_factor = (R - min_fraction * r) / (1 - min_fraction)
 
     return (m.e_pro_out[tm, stf, sit, pro, coo] ==
-            m.dt * m.cap_pro[stf, sit, pro] * online_factor +
+            m.dt * (m.cap_pro[stf, sit, pro] * online_factor +
             m.tau_pro[tm, stf, sit, pro] * throughput_factor)
```
Comment (Dthushara246) [inline urbs/model.py:633]: Another closing bracket at the end is needed.

File: doc/implementdoc/equ-commodity.rst (hunk 2 above)
Comment (Dthushara246) [reply doc/implementdoc/equ-commodity.rst:416]: Yes. So line 416 should not be updated with addition of 'y'.

**Pull request merged**